WcaOnRails/app/webpacker/javascript/edit-events/EditEvents.js

```diff
@@ -154,6 +154,9 @@ function RoundsTable({ wcifEvents, wcifEvent, disabled }) {
           })}
         </tbody>
       </table>
+      <h5>
+        Qualification: <EditQualificationButton wcifEvent={wcifEvent} disabled={disabled} />
```

Comment:
`Qualification: ` prefix should be handled via i18n

Reply:
Good point, but that shouldn't keep us from improving localization one step at a time :)
Especially for cases where we already have translations in the existing YML, that's a very low hanging fruit.